.gitignore

```diff
@@ -6,3 +6,4 @@ target
 /.project
 /.settings
 /RUNNING_PID
+.DS_Store
```

Comment:
- You probably want to add this to your global .gitignore
- Also, there's no new line at the end of the file